trace-sdk/src/main/java/io/bitrise/trace/TraceSdk.java

```diff
@@ -41,6 +41,11 @@
      */
     public static final String NAME = "Trace Android";
 
+    /**
+     * Flag to configure the sdk into debug mode, by default this will be false.
+     */
+    public static boolean DEBUG_ENABLED = false;
```

Comment:
I would recommend making this private, and access/mutate only with the getter/setter.

Reply:
Have updated - thank you.